java/compat/src/main/java/com/microsoft/bond/compat/CompatDriver.java

```diff
@@ -58,10 +58,9 @@ public static void main(final String[] args) throws IOException {
             System.exit(STATUS_FAILURE);
         }
 
-        final Compat compat = new Compat();
         final Serializer<Compat> serializer = new Serializer<>();
-        final Deserializer<Compat> deserializer = new Deserializer<>((StructBondType<Compat>) compat.getBondType());
-        deserializer.deserialize(reader);
+        final Deserializer<Compat> deserializer = new Deserializer<>((StructBondType<Compat>) Compat.BOND_TYPE);
```

Comment:
Oh, I wasn't even looking at that - got thrown by the static vs. instance ways of getting bond types. No problem here.